docs/application/web/get-started/web-service/first-service.md

```diff
@@ -0,0 +1,156 @@
+# Create Your First Web Service Application
+
+**Welcome to Web Service Application Development!**
+
+A web serivce is a JavaScript business logic running in node.js environment with a config.xml file in tizen package.
+It means that you can also use tizen web service life-cycle and node.js built-in modules.
+
+Take a look at the following instructions to help familiarize yourself
+with the Tizen [Web application development process](../../tutorials/process/app-dev-process.md) as well as
+using the Tizen Studio and installing the created application on the emulator or target device.
+The web service is a kind of web application and is similar with the web application in developing process.
+With these instructions, you can create and run a basic web service, which has web service life-cycle and communications
+between web service and web application.
+
+1.  Before you get started with developing web service application, download and install the [Tizen Studio](../../../tizen-studio/index.md)
+
+    For more information on the installation process, see the [installation guide](../../../tizen-studio/setup/install-sdk.md).
+
+2.  [Create a web service project](#create) using the Tizen Studio
+
+    This step shows how you can use a predesigned project template that creates all the basic files and folders, required for your project.
+
+3.  [Build the web service](#build)
+
+    After you have implemented code for the features you want, this step shows how you can build the web service to validate and compile the code.
+
+4.  [Run the web service](#run)
+
+    This step shows how you can run the web service on an emulator or tizen target devices.
+
+5.  [Design a UI](#ui)
```

Comment:
Because service app doesn't support UI, please change this phase to other content.

You had better add how to use node.js or add how to edit a code to use a general function such as using a DOM.

Reply:
@journey2w  Sure, wont post redundant comments in future. I just referred to the developer to ensure to do what you said.

Reply:
> Because service app doesn't support UI, please change this phase to other content.
> 
> You had better add how to use node.js or add how to edit a code to use a general function such as using a DOM.

The web service template is newly released. I will update it.